src/Lecture4.hs

```diff
@@ -237,7 +315,24 @@ you can return string "no value"
 -}
 
 displayStats :: Stats -> String
-displayStats = error "TODO"
+displayStats stats = "Total positions:       : " ++
```

Comment:
You can use standard function `unlines` and put all lines as separate elements of the list 👍🏻 
It can make the implementation a bit cleaner as you at least don't need to hardcode `\n` in lines.